kotlin/configure-subnetwork-trace/README.md

```diff
@@ -20,12 +20,12 @@ Example barrier conditions for the default dataset:
 
 1. Populate the traversability scope list with the enum values from `UtilityTraversabilityScope`.
 2. Create and load a `UtilityNetwork` with a feature service URL, then get an asset type and a tier by their names.
-3. Populate the choice list for the comparison source with the non-system defined `Definition.networkAttributes`.  Populate the choice list for the comparison operator with the enum values from `UtilityAttributeComparisonOperator`. 
+3. Populate the choice list for the comparison source with the non-system defined `Definition.networkAttributes`.  Populate the choice list for the comparison operator with the enum values from `UtilityAttributeComparisonOperator`.
 4. Create a `UtilityElement` from this asset type to use as the starting location for the trace.
 5. Update the selected barrier expression and the checked options in the UI using this tier's `TraceConfiguration`.
 6. When 'Network Attribute' is selected, if its `Domain` is a `CodedValueDomain`, populate the choice list for the comparison value with its `CodedValues`.  Otherwise, display a free-form textbox for entering an attribute value.
```

Comment:
I think we could improve this instruction since there is no class called just "Domain" used in this sample
```suggestion
6. When 'Network Attribute' is selected, check whether it has a `CodedValueDomain`, and if so, populate the choice list for the comparison value with its `CodedValues`.  Otherwise, display a free-form textbox for entering an attribute value.
```